src/core/qgsaggregatecalculator.cpp

```diff
@@ -42,13 +42,20 @@ void QgsAggregateCalculator::setParameters( const AggregateParameters &parameter
   mOrderBy = parameters.orderBy;
 }
 
+void QgsAggregateCalculator::setFidsFilter( const QgsFeatureIds fids )
```

Comment:
const ref
```suggestion
void QgsAggregateCalculator::setFidsFilter( const QgsFeatureIds &fids )
```

Reply:
An oversight on my part, should be fixed now, thank you for the review!